### PR TITLE
Change the license to the new BSD-style license

### PR DESCRIPTION
**Pull request by jasondegraw**
Remove the old notices from the end of files and place the new notice and license at the top of the files.

Sadly, I have no unit test. I am ashamed.

### CODE REVIEW

**Comment by JasonGlazer** [comment]
@jasondegraw Probably too late to ask, why can't these be at the end of the files instead of at the top?

**Comment by jasondegraw** [comment]
@JasonGlazer It's not too late. I know that placing the notice/license at the top may be inconvenient depending on the editor used. I placed it at the top because that's a more typical placement and there were 100+ files in the repo had no notice. It is easier to automatically find the notice when it's placed at the beginning, so going forward it'll be easier to make sure that everything has the correct text.

**Comment by JasonGlazer** [comment]
I am not sure how others feel about it but boilerplate, while necessary, is kind of in the way of getting code done so it is better at the bottom of files.

**Comment by mjwitte** [comment]
@jasondegraw Perhaps using a longer line width would make it less obtrusive at the top?

**Comment by Myoldmopar** [comment]
It is _very_ standardized in software to put the license text at the top of the file.  Let's keep it at the top.

File: src/EnergyPlus/AirTerminalUnit.hh [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+// If you have questions about your rights to use or distribute this software,
+// please contact Berkeley Lab's Innovation & Partnerships Office at
+// IPO@lbl.gov.
+//
+// NOTICE: This Software was developed under funding from the U.S. Department
+// of Energy and the U.S. Government consequently retains certain rights. As
+// such, the U.S. Government has been granted for itself and others acting on
+// its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
+// Software to reproduce, distribute copies to the public, prepare derivative
+// works, and perform publicly and display publicly, and to permit other to do
```
Comment (mjwitte) [inline src/EnergyPlus/AirTerminalUnit.hh]: Should "other" be "others"?

Comment (jasondegraw) [inline src/EnergyPlus/AirTerminalUnit.hh]: Yes, I would think so.

**Comment by jasondegraw** [comment]
@JasonGlazer @mjwitte I agree that it would be better if the lines were longer. I arbitrarily picked 80 columns, changing it to 100 shortens it by 11 lines (from 68 to 57). How does that sound?

**Comment by Myoldmopar** [comment]
Don't go past [140](https://dev.twitter.com/overview/api/counting-characters) or teh internetz breaks!!

**Comment by mjwitte** [comment]
Sure, that works.

**Comment by Myoldmopar** [comment]
OK, I'm merging this shortly simultaneously as the website gets updated.  Get ready...

**Comment by mjwitte** [comment]
@Myoldmopar Do we want to tag this as NewFeature or DoNotPublish?

**Comment by Myoldmopar** [comment]
DoNotPublish.

On Thu, Feb 11, 2016 at 9:12 AM, Michael J. Witte notifications@github.com
wrote:

> @Myoldmopar https://github.com/Myoldmopar Do we want to tag this as
> NewFeature or DoNotPublish?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/NREL/EnergyPlus/pull/5329#issuecomment-182938437.

## 

Edwin Lee, PhD
Engineer, Commercial Buildings Research Staff
National Renewable Energy Laboratory

**Pull request merged**